bridge/server/interfaces/webhook-config-yaml.ts

```diff
@@ -69,9 +75,15 @@ export class WebhookConfigYaml implements WebhookConfigYamlResult {
           requests: [curl],
           envFrom: secrets,
           subscriptionID: subscriptionId,
+          sendFinished: sendFinished,
         });
       } else {
-        this.spec.webhooks.push({ type: eventType, requests: [curl], subscriptionID: subscriptionId });
+        this.spec.webhooks.push({
```

Comment:
Because the information here was updated several times in the past, I would change the if else (`secrets.length`) to 
```ts
this.spec.webhooks.push({
  type: eventType,
  requests: [curl],
  ...(secrets.length  && { envFrom: secrets }),
  subscriptionID: subscriptionId,
  sendFinished: sendFinished,
});
```
to be less redundant.